CBLClient/Apps/CBLTestServer-C/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,29 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.couchbase.testserver">
```

Comment:
Can you please make sure Android app name is as com.couchbase.TestServerApp

Reply:
Is this something you cannot do?  I don't see why it matters but if you want to change it you are welcome to.